assets/js/blocks/attribute-filter/block.js

```diff
@@ -168,19 +168,26 @@ const AttributeFilterBlock = ( { attributes } ) => {
 		setCheckedOptions( checked );
 	}, [] );
 
-	if ( ! taxonomy ) {
+	if ( ! taxonomy || ( options.length === 0 && ! attributeTermsLoading ) ) {
 		return null;
 	}
 
+	const TagName = `h${ attributes.headingLevel }`;
+
 	return (
-		<div className="wc-block-attribute-filter">
-			<CheckboxList
-				className={ 'wc-block-attribute-filter-list' }
-				options={ options }
-				onChange={ onChange }
-				isLoading={ attributeTermsLoading }
-			/>
-		</div>
+		<Fragment>
+			{ ! isPreview && attributes.heading && (
```

Comment:
If by default we are displaying the header, shouldn't we display it in the preview too?

Reply:
I did this so we would display a proper heading tag on the frontend, but a PlainText element in the editor - that is rendered in edit.js to avoid adding to the block itself.

Reply:
Oh, ok, it means it's in the editor. I thought it meant the inserter preview.

I see `isPreview` is used in other parts of the code base, so I'm fine merging this PR as-is, but how does it sound changing the name of this prop everywhere to something like `inEditor`, `editorView` or something similar? I can take care of this, but I find using the word _preview_ is confusing with the inserter preview added in WP 5.3.

Reply:
Yeah I agree it's confusing now.